addons/account/models/account_move.py

```diff
@@ -222,8 +222,8 @@ def _get_default_invoice_incoterm(self):
         string='Vendor Bill',
         help="Auto-complete from a past bill.")
     invoice_source_email = fields.Char(string='Source Email', tracking=True)
-    invoice_vendor_display_name = fields.Char(compute='_compute_invoice_vendor_display_info', store=True)
-    invoice_vendor_icon = fields.Char(compute='_compute_invoice_vendor_display_info', store=False)
+    invoice_partner_display_name = fields.Char(compute='_compute_invoice_vendor_display_info', store=True)
+    invoice_partner_icon = fields.Char(compute='_compute_invoice_vendor_display_info', store=False)
```

Comment:
I think the name of the compute could be renamed like `_compute_invoice_partner_display_info`?

Reply:
Yes sir!